extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
```

Comment:
I am not convinced it needs an experiment, it is not changing current behaviour and it does not have a large API surface that we expect to change. Experiment flags are good in general but there are cons to adding them for every little featue. Like forgetting to be cleaned up (which happened to amp-accordion's `disable-session-states` attribute)